README.md

```diff
@@ -25,46 +47,42 @@ From there, we’ll design wireframes for each app, composing them from the list
 
 See the [IPFS GUI Project Description](https://docs.google.com/document/d/1HzwTYo4BDDH4WIh0EULh0U9_WnT84FacDUdVtTExluQ/edit?usp=sharing) document for the original definition of this project.
 
-## The IPFS GUI Triangle
-
-```
-
-                              Webui
-                                Λ
-                               ╱ ╲
-                              ╱   ╲
-                             ╱     ╲
-                            ╱       ╲
-                           ╱         ╲
-                          ╱           ╲
-                         ╱             ╲
-                        ╱               ╲
-                       ╱                 ╲
-                      ╱                   ╲
-                     ╱┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ┐╲
-                    ╱   IPFS GUI Triangle   ╲
-                   ╱  └ ─ ─ ─ ─ ─ ─ ─ ─ ─ ┘  ╲
-                  ╱                           ╲
-                 ╱                             ╲
-                ╱                               ╲
-     Companion  ───────────────────────────────── Desktop
-```
-
-
-## Docs, Assets and Code
-
-- PM
+## Resources
+
+#### Assets
+  - IPFS Logo - [repo](https://github.com/ipfs/logo) - _vector and raster logo assets_
+  - IPFS brand book - [_pdf_](https://github.com/ipfs-shipyard/ipfs-ui-style-guide/files/1629262/IPFS_brandbook.pdf) - _how to use the IPFS logo_
+  - IPFS UI kit - [_png_](img/ipfs-ui-kit.png) - _wip on a UI style guide_
+  - Web UI Wireframes - [_invision_](https://app.zeplin.io/project/5a32d45d1a17248135241058)
+  - Design Tools Research - [google doc](https://docs.google.com/document/d/1qJyfwgcMg8l3Tk3aYxF38iyYRhkEf3nlLNqOw4ZiW_8/edit)
+
+#### PM
+
   - [Project Work Plan & Description](https://docs.google.com/document/d/1HzwTYo4BDDH4WIh0EULh0U9_WnT84FacDUdVtTExluQ/edit#heading=h.a415cvyt09h4)
   - [Main Google Drive folder](https://drive.google.com/drive/u/1/folders/1xu_lv1jsatKnwyFcjd_fDsg3rCi9550u)
   - [Research](research)
 
-- Design
-  - Assets in Google Drive (soon™)
-  - Zeplin
-    - [WebUI Wireframe](https://app.zeplin.io/project/5a32d45d1a17248135241058)
-  - [Design Tools Research](https://docs.google.com/document/d/1qJyfwgcMg8l3Tk3aYxF38iyYRhkEf3nlLNqOw4ZiW_8/edit)
+## Team
+
+In alphabetical order we are:
+
```

Comment:
Should we link these to the GitHub profiles? 🤔

Reply:
i considered it, but then left it. If people wanna creep around our profiles they can figure it out, but i don't feel strongly about it, happy to link them if people feel positive about it

Reply:
Eh, don't have a strong opinion on that too 🤷‍♂️